### PR TITLE
Have Redis provide StateRegionConfig

### PR DESCRIPTION
**Pull request by tomerovadia**
Right now, every time we want to update the StateRegionConfig we have to go through the normal release cycle, which takes time, testing and often needs syncing with whatever other code changes are happening.

This PR instead has Redis provide the StateRegionConfig, so that any changes are easily made much more quickly via Redis, not requiring a release.

### CODE REVIEW

File: src/state_region_config.ts [hunk 1]
```diff
@@ -1,67 +1,34 @@
...
+  );
+  if (stateRegionConfig) return stateRegionConfig;
+  return {} as StateRegionConfig;
+}
+
+export async function regionsList(
+  redisClient: PromisifiedRedisClient
+): Promise<string[]> {
+  return process.env.CLIENT_ORGANIZATION === 'VOTE_AMERICA'
+    ? uniq(Object.values(fetchStateRegionConfig(redisClient)))
```
Comment (fongandrew) [inline src/state_region_config.ts]: We can probably remove the client org check if it's in Redis anyway?

**Review by fongandrew — APPROVED** [review]

File: src/state_region_config.ts (hunk 1 above)
Comment (tomerovadia) [reply src/state_region_config.ts]: The other client uses a separate Redis. Do you know if I would need to update their Redis as well for this to work properly?

Comment (fongandrew) [reply src/state_region_config.ts]: It should? I assume it's an empty hash on the other client, so this function will just return an empty list.

Comment (tomerovadia) [reply src/state_region_config.ts]: Good point. I'll test that out and follow up.

Comment (tomerovadia) [reply src/state_region_config.ts]: It works! Thanks for suggesting that. I pushed a commit with the change, as well as another small but significant change to how fallbacks work if Redis is lacking needed information. Please take a look

File: src/state_region_config.ts [hunk 2]
```diff
@@ -21,9 +21,10 @@ export async function fetchStateRegionConfig(
 export async function regionsList(
   redisClient: PromisifiedRedisClient
 ): Promise<string[]> {
-  return process.env.CLIENT_ORGANIZATION === 'VOTE_AMERICA'
-    ? uniq(Object.values(fetchStateRegionConfig(redisClient)))
-    : [];
+  const stateRegionConfig = await fetchStateRegionConfig(redisClient);
+  console.log("TOMERTOMERTOMERTOMERTOMERTOMEROMTERTOMERTOMERTOMER");
```
Comment (fongandrew) [inline src/state_region_config.ts]: Stray console log

**Comment by fongandrew** [comment]
Just the stray log but everything else looks fine to me.

File: src/state_region_config.ts (hunk 2 above)
Comment (tomerovadia) [reply src/state_region_config.ts]: Thanks! Removed

**Review by fongandrew — APPROVED** [review]

**Pull request merged**